tools/docker/docker-entrypoint.sh

```diff
@@ -54,6 +54,17 @@ else
   echo "[INFO] No custom assets found at ${CUSTOM_ASSETS_PATH}"
 fi
 
+# Executing custom scripts located in CUSTOM_SCRIPTS_DIRECTORY if environment variable is set
+if [[ -z "${CUSTOM_SCRIPTS_DIRECTORY}" ]]; then
+  echo "[INFO] No CUSTOM_SCRIPTS_DIRECTORY env variable set"
+else
+  echo "[INFO] CUSTOM_SCRIPTS_DIRECTORY env variable set to ${CUSTOM_SCRIPTS_DIRECTORY}"
+  chmod +x ${CUSTOM_SCRIPTS_DIRECTORY}/*.sh  
```

Comment:
I wouldn't do that here, the scripts should be made executable before running the docker image in my opinion. Modifying the file (even just its rights) from a docker run is a little bit risky; what if it fails?

Reply:
agree here